archive_query_log/orm.py

```diff
@@ -190,6 +190,18 @@ class Snippet(SnippetId):
     text: str | None = Text()
 
 
+class DirectAnswerId(InnerDocument):
+    id: str = Keyword()
+    rank: int = Integer()
+
+
+class DirectAnswer(DirectAnswerId):
+    content: str = Text()
+    big_box: str | None = Keyword()
+    small_box: str | None = Text()
+    right_box: str | None = Text()
```

Comment:
Yeah, I think one direct answer object should correspond to one "box".
And then, I would favor storing the HTML code of the box's HTML element (`content`), the answer text (`text`), and maybe the URL where the answer links to (`url`).